Dockerfile

```diff
@@ -794,6 +794,7 @@ RUN <<END
 END
 
 FROM rootfs-base-${TARGETARCH} AS rootfs-base
+RUN mkdir -p /rootfs/usr/etc
```

Comment:
https://github.com/dsseng/talos/blob/8680351c131d29a76682569742dbd44c8ffe47d3/Dockerfile#L701 this should have been here, can fixup later

Reply:
Oops, will do. Thanks!